django_stomp/execution.py

```diff
@@ -22,6 +22,7 @@
 wait_to_connect = int(getattr(settings, "STOMP_WAIT_TO_CONNECT", 10))
 durable_topic_subscription = eval_str_as_boolean(getattr(settings, "STOMP_DURABLE_TOPIC_SUBSCRIPTION", False))
 listener_client_id = getattr(settings, "STOMP_LISTENER_CLIENT_ID", None)
+is_correlation_id_required = getattr(settings, "STOMP_CORRELATION_ID_REQUIRED", True)
```

Comment:
Following the line 23, you can use like this:
```suggestion
is_correlation_id_required = eval_str_as_boolean(getattr(settings, "STOMP_CORRELATION_ID_REQUIRED", True))
```